e2e/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "allowUnreachableCode": false,
     "suppressExcessPropertyErrors": false,
     "moduleResolution": "node",
-    "module": "commonjs",
+    "module": "ES2015",
```

Comment:
`jest.config.js`: `const { defaults } = require('jest-config');` throws tslib error without this change.